packages/opencensus-core/src/metrics/gauges/derived-gauge.ts

```diff
@@ -94,12 +95,19 @@ export class DerivedGauge implements types.Meter {
    * @param {string} unit The unit of the metric.
    * @param {MetricDescriptorType} type The type of metric.
    * @param {LabelKey[]} labelKeys The list of the label keys.
+   * @param {Map<LabelKey, LabelValue>} constantLabels The map of constant
+   *     labels for the Metric.
    */
   constructor(
       name: string, description: string, unit: string,
-      type: MetricDescriptorType, labelKeys: LabelKey[]) {
-    this.metricDescriptor = {name, description, unit, type, labelKeys};
+      type: MetricDescriptorType, labelKeys: LabelKey[],
+      readonly constantLabels: Map<LabelKey, LabelValue>) {
     this.labelKeysLength = labelKeys.length;
+    const combinedLabelKeys = labelKeys.concat(...constantLabels.keys());
```

Comment:
I notice below you use `[...gaugeEntry.labelValues, ...this.constantLabelValues]`, so should this be `[...labelKeys, ...consantLabels.keys()]` to match?

Reply:
Good Idea, Done.